### PR TITLE
AF-1532 Replaced Chronicle Queue with chronicle Map to expose the log of Maven to the UI

### PR DESCRIPTION
**Pull request by desmax74**
Replaced Chronicle Queue with chronicle Map to expose the log of Maven to the UI

Bootstrap: https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/881

### CODE REVIEW

File: kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-offprocess-core/pom.xml [hunk 1]
```diff
@@ -30,6 +30,11 @@
         <artifactId>kie-wb-common-compiler-testutil</artifactId>
         <version>${version.org.kie}</version>
       </dependency>
+      <dependency>
+        <groupId>net.openhft</groupId>
+        <artifactId>chronicle-map</artifactId>
+        <version>3.16.4</version>
```
Comment (jomarko) [inline kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-offprocess-core/pom.xml]: I think we need to use a managed version here.

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by domhanak** [comment]
@desmax74 is this still relevant?

**Comment by desmax74** [comment]
No more relevant

**Pull request closed without merging**